bazel/grpc_deps.bzl

```diff
@@ -14,6 +14,7 @@
 """Load dependencies needed to compile and test the grpc library as a 3rd-party consumer."""
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "new_git_repository")
```

Comment:
remove this line (and use http_archive instead).

Reply:
done